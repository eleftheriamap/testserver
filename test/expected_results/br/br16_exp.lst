
test/exp/test/gen/br/br16_exp.bin:     file format binary


Disassembly of section .data:

0000000000000000 <.data>:
   0:	d285b5bb 	mov	x27, #0x2dad                	// #11693
   4:	f2a0009b 	movk	x27, #0x4, lsl #16
   8:	d280001a 	mov	x26, #0x0                   	// #0
   c:	f2b1401a 	movk	x26, #0x8a00, lsl #16
  10:	f800077a 	str	x26, [x27], #0
  14:	d61f0360 	br	x27
  18:	8a000000 	and	x0, x0, x0
