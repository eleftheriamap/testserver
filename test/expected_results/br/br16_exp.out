Registers:
X00    = 0000000000000000
X01    = 0000000000000000
X02    = 0000000000000000
X03    = 0000000000000000
X04    = 0000000000000000
X05    = 0000000000000000
X06    = 0000000000000000
X07    = 0000000000000000
X08    = 0000000000000000
X09    = 0000000000000000
X10    = 0000000000000000
X11    = 0000000000000000
X12    = 0000000000000000
X13    = 0000000000000000
X14    = 0000000000000000
X15    = 0000000000000000
X16    = 0000000000000000
X17    = 0000000000000000
X18    = 0000000000000000
X19    = 0000000000000000
X20    = 0000000000000000
X21    = 0000000000000000
X22    = 0000000000000000
X23    = 0000000000000000
X24    = 0000000000000000
X25    = 0000000000000000
X26    = 000000008a000000
X27    = 0000000000042dad
X28    = 0000000000000000
X29    = 0000000000000000
X30    = 0000000000000000
PC     = 0000000000042dad
PSTATE : -Z--
Non-zero memory:
0x00000000 : 0xd285b5bb
0x00000004 : 0xf2a0009b
0x00000008 : 0xd280001a
0x0000000c : 0xf2b1401a
0x00000010 : 0xf800077a
0x00000014 : 0xd61f0360
0x00000018 : 0x8a000000
0x00042db0 : 0x0000008a
